Error on received attestation

### Description
Got a series of errors like this one on Onyx:

```
"class":"AbstractRouter","message":"Exception while handling message from peer io.libp2p.etc.util.P2PServiceSemiDuplex$SDPeerHandler@16f6ba84: from: "\000%\b\002\022!\002S\267o\023\214B/\237[E\302\2059\3548\254\001\375i\345\270\177\223\304\177\036-B\215S)R"
data: "\365\001\030\344\000\000\000\200\001\0002\001\000\200\202\246\3671\274\266\332\032u\346\314\365\376\262\301 <\233\004\024x\301Z\265\336{\005\177\363I~\234\v\r.\200/\2008n\260jS\372m\343\233\r\250\311\2410\251\354\227ka\002\275?Z\365\302s\252\005\272\311\f\r(~P\000\360p\220\020\220\317G-\344&6\f\025\305e\224qm\031\233\307\024sd\302\027kV9TO\347hG->\261\342\302\347\240\021I\221\275\033^\364\271{\n\035T#\324\330A%T\336\375\366\266\"\036\224\3241\244\3401HA\376q\305w\323\265a!\356X\r\353\357\374\004\366\201s\365\363n\315+\223\215\000\200\000\000\000\000\000\000\000\000\000\000\000\000\000\000\001"
seqno: "\026\030L\034E?\226\374"
topicIDs: "/eth2/a65b4897/beacon_attestation_0/ssz_snappy"
","throwable":" java.util.concurrent.ExecutionException: java.lang.UnsupportedOperationException: Empty collection can't be reduced.
at java.base/java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:395)
at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:1999)
at io.libp2p.pubsub.AbstractRouter.onInbound(AbstractRouter.kt:209)
at io.libp2p.etc.util.P2PService.streamInbound(P2PService.kt:149)
at io.libp2p.etc.util.P2PService$StreamHandler$channelRead$1.invoke(P2PService.kt:49)
at io.libp2p.etc.util.P2PService$StreamHandler$channelRead$1.invoke(P2PService.kt:35)
at io.libp2p.etc.util.P2PService$sam$java_lang_Runnable$0.run(P2PService.kt)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)\nCaused by: java.lang.UnsupportedOperationException: Empty collection can't be reduced.
at io.libp2p.pubsub.PubsubApiImpl$onNewMessage$1.invoke(PubsubApiImpl.kt:130)
at io.libp2p.pubsub.PubsubApiImpl$onNewMessage$1.invoke(PubsubApiImpl.kt:23)
at io.libp2p.etc.types.AsyncExtKt$thenApplyAll$1.apply(AsyncExt.kt:84)
at io.libp2p.etc.types.AsyncExtKt$thenApplyAll$1.apply(AsyncExt.kt)
at java.base/java.util.concurrent.CompletableFuture.uniApplyNow(CompletableFuture.java:680)
at java.base/java.util.concurrent.CompletableFuture.uniApplyStage(CompletableFuture.java:658)
at java.base/java.util.concurrent.CompletableFuture.thenApply(CompletableFuture.java:2094)
at io.libp2p.etc.types.AsyncExtKt.thenApplyAll(AsyncExt.kt:79)
at io.libp2p.pubsub.PubsubApiImpl.onNewMessage(PubsubApiImpl.kt:73)
at io.libp2p.pubsub.PubsubApiImpl.access$onNewMessage(PubsubApiImpl.kt:23)
at io.libp2p.pubsub.PubsubApiImpl$1.invoke(PubsubApiImpl.kt:55)
at io.libp2p.pubsub.PubsubApiImpl$1.invoke(PubsubApiImpl.kt:23)
at io.libp2p.pubsub.AbstractRouter.onInbound(AbstractRouter.kt:194)\n\t... 10 more\n"}
```

--- a/gradle/versions.gradle
+++ b/gradle/versions.gradle
@@ -27,7 +27,7 @@ dependencyManagement {
     dependency 'io.protostuff:protostuff-core:1.6.2'
     dependency 'io.protostuff:protostuff-runtime:1.6.2'
 
-    dependency 'io.libp2p:jvm-libp2p-minimal:0.5.1-RELEASE'
+    dependency 'io.libp2p:jvm-libp2p-minimal:0.5.2-RELEASE'
     dependency 'org.jetbrains.kotlin:kotlin-stdlib:1.3.61'
 
     dependency 'io.pkts:pkts-core:3.0.3'

